automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookExtensionSpec.scala

```diff
@@ -87,13 +88,14 @@ class NotebookExtensionSpec extends ClusterFixtureSpec with NotebookTestUtils {
               notebookPage.addCodeAndExecute("1+1")
 
               notebookPage.saveNotebook()
-              //sleep 4 minutes. We do this to ensure the assertions are true after a certain about of time
+
+              //sleep 4 minutes. We do this to ensure the assertions are true after a certain about of time.
+              //eventually is not applicable as we want to ensure its true after the time period, not at some point until timeout
               logger.info("Waiting 4 minutes as lock takes time to be reflected in metadata")
               Thread.sleep(240000)
```

Comment:
can we remove this sleep now?

Reply:
I don't think we can. I added an additional comment to explain why it's needed. The goal is to check that after 4minutes, the condition holds true. Eventually only checks until it is true. I looked through the ```org.scalatest.concurrent``` documentation and couldn't find any cleaner ways to do this.